# Example configuration:
# - velocity limits are around a 10% above the physical limits
# - acceleration limits are just low enough to avoid jerking

yocs_velocity_smoother:
  speed_lim_v: 1.5
  speed_lim_w: 1.0
  
  accel_lim_v: 0.2
  accel_lim_w: 1.0
  
  frequency: 50.0
  decel_factor: 50
  
  robot_feedback: 0
